recipe/meta.yaml

```diff
@@ -51,6 +53,18 @@ requirements:
     - pywin32  # [win64]
 
 test:
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
Unlike in the CUDA 12 case, where the C/C++ compilers are brought in via `cuda-nvcc`, for CUDA 11 we need to explicitly request a host compiler here.

I also noticed that if we don't do this, on Windows the MSVC from the VM would be invoked.

Reply:
We can just skip the test on Windows CUDA 11 for now and follow up separately